### ISSUE TITLE
Redis err when trying to perform finding operations (One or more score can't be converted into double)

### ISSUE THREAD

**Issue by tyronedougherty** [biggora/caminte#92]
Just starting out with the framework, for some reason trying to get models from the database is leading to errors. If I use anything else apart from find by ID, for example:
```
User.all(function(err, users){
   console.log(users);
});
```
I receive this error:
```
[Error: ERR One or more scores can't be converted into double] command: 'SORT', code: 'ERR'
Error: ERR One or more scores can't be converted into double
    at JavascriptReplyParser.parseResult (node_modules/redis-parser/lib/javascript.js:90:16)
    at JavascriptReplyParser.tryParsing (node_modules/redis-parser/lib/javascript.js:117:21)
    at JavascriptReplyParser.run (node_modules/redis-parser/lib/javascript.js:131:22)
    at JavascriptReplyParser.execute (node_modules/redis-parser/lib/javascript.js:112:10)
    at Socket.<anonymous> (node_modules/redis/index.js:223:27)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:523:20)
```

Any idea what's going on here?
My redis is 2.5.3 and caminte is 0.2.7

My model looks like this:
```
const uuid = require('uuid');

let User = schema.define('User', {
    id: {type: schema.String, default: uuid.v4, unique: true},
    email: {type: schema.String, "null":false, unique: true},
    name: {type: schema.String},
    test: schema.Boolean
}, {
    primaryKeys: ['id']
});

User.validatesPresenceOf('id', 'name', 'email');
```

Any help is much appreciated :)

**Comment by tyronedougherty** [comment]
Upon further inspection it seems that this error is caused by using uuids instead of integers as the primary key because it screws with the scoring the redis sort function somehow. 

Anyone got any idea how to enable using uuids as the primary key? Many thanks :)

**Comment by tyronedougherty** [comment]
I've found the issue and have fixed it in pull request #93 
Hopefully it is approved for merge :)
